lib/sidekiq/web/application.rb

```diff
@@ -82,11 +82,14 @@ def self.set(key, val)
 
     post "/busy" do
       if params["identity"]
-        p = Sidekiq::Process.new("identity" => params["identity"])
-        p.quiet! if params["quiet"]
-        p.stop! if params["stop"]
+        pro = processes.find { |p| p.identity == params["identity"] }
```

Comment:
Is there a way to get a single `Sidekiq::Process` by identity efficiently instead of loading the whole process list to get one? I figured this was fine anyway as the page you're on shows the whole list.

Reply:
This has been added. From what I can tell, the key that goes in `processes` is actually the identity but I would like to confirm this, as some of the tests use a different key to the identity they have in their info.